include/sentry.h

```diff
@@ -338,26 +338,86 @@ typedef enum sentry_level_e {
 } sentry_level_t;
 
 /**
- * Creates a new empty event value.
+ * Creates a new empty Event value.
+ *
+ * See https://develop.sentry.dev/sdk/event-payloads/
```

Comment:
Thank you for linking into documentation. `develop.sentry.dev` is mainly targeted at SDK engineers. Should we rather link to relevant sections in `docs.sentry.io` that explain these concepts for users?

For the native SDK, it definitely makes sense to keep a secondary link to develop, since we do not have a typed out event schema.

cc @bruno-garcia @HazAT

Reply:
Since developers need to construct event objects themselves, I think its best to link to the page that defines the possible properties.

Reply:
Agreed, that's why I'm proposing to link _both_, with product documentation being the primary link. In other SDKs, we would skip the develop link.